curriculum/challenges/english/25-front-end-development/quiz-semantic-html/66ed903cf45ce3ece4053ebe.md

```diff
@@ -1384,3 +1384,482 @@ Which element is used to represent subscript text?
+
+### --question--
+
+#### --text--
+
+What does it mean when an HTML element is considered "non-semantic"?
+
+#### --distractors--
+
+It automatically applies default browser styles.
+
+---
+
+It provides no information about the meaning or purpose of its content.
+
+---
+
+It is only used within forms and interactive elements.
+
+#### --answer--
+
+It means the element does not convey any inherent meaning about its content; it is used solely for grouping or styling.
+
+### --question--
+
+#### --text--
+
+What is the role of the `rt` element within a ruby annotation?
+
+#### --distractors--
+
+It encloses the base text that is being annotated.
+
+---
+
+It provides fallback text if ruby annotations are not supported.
+
+---
+
+It emphasizes the wrapped text.
+
+#### --answer--
+
+It specifies the annotation text (such as pronunciation or meaning) that appears above or below the base text in a ruby annotation.
+
+### --question--
+
+#### --text--
+
+Which of the following represents a proper way of defining a foreign language phrase?
+
+#### --distractors--
+
+```html
+  <i lng="fr">je ne sais quoi</i>
+```
+
+---
+
+```html
+  <i language="fr">je ne sais quoi</i>
+```
+
+---
+
+```html
+  <i lg="fr">je ne sais quoi</i>
+```
+
+#### --answer--
+
+```html
+  <i lang="fr">je ne sais quoi</i>
+```
+
+### --question--
+
+#### --text--
+
+Which element will be used to display text that has strong importance?
+
+#### --distractors--
+
+`em`
+
+---
+
+`b`
+
+---
+
+`sub`
+
+#### --answer--
+
+`strong`
+
+### --question--
+
+#### --text--
+
+Which element is used to annotate spelling errors?
+
+#### --distractors--
+
+`strong`
+
+---
+
+`em`
+
+---
+
+`s`
+
+#### --answer--
+
+`u`
+
+### --question--
+
+#### --text--
+
+Which element would be useful for marking a phrase from another language?
+
+#### --distractors--
+
+`em`
+
+---
+
+`u`
+
+---
+
+`s`
+
+#### --answer--
+
+`i`
+
+### --question--
+
+#### --text--
+
+What would be a common use case for the `sup` element?
+
+#### --distractors--
+
+To represent a phrase from another language.
+
+---
+
+To represent contact information.
+
+---
+
+To represent a section that is quoted from another source.
+
+#### --answer--
+
+To represent exponents, superior lettering and ordinal numbers.
+
+### --question--
+
+#### --text--
+
+Why might a developer choose a semantic element over a generic `div` element?
+
+#### --distractors--
+
+Semantic elements load faster than `div` elements.
+
+---
+
+Semantic elements inherently provide meaning, which improves readability and accessibility.
+
+---
+
+Generic `div` elements are not supported on mobile devices.
+
+#### --answer--
+
+Because semantic elements add meaning to the code, making it more understandable for both developers and assistive technologies.
+
+### --question--
+
+#### --text--
+
+What is the purpose of the `footer` element in an HTML document?
```

Comment:
This question should be replaced as the topic was covered in the Basic HTML section.